bin/sl-deploy.js

```diff
@@ -62,15 +65,27 @@ if (numArgs > 2) {
   process.exit(1);
 }
 
+var workingDir = process.cwd();
+
+var packageInfo = getPackageInfo(workingDir);
+serviceName = serviceName || (packageInfo ? packageInfo.name : null);
+
+if (!serviceName) {
+  console.error(
+    'Unable to detect service name from package. ' +
+    'Please specify a service name. Try `%s --help`.', $0
```

Comment:
alternatively, this could be `specify a service name or add a "name" property to your package.json. Try...`